sql-plugin/src/main/scala/org/apache/spark/sql/rapids/BasicColumnarWriteStatsTracker.scala

```diff
@@ -206,6 +219,7 @@ class BasicColumnarWriteJobStatsTracker(
     var numFiles: Long = 0L
     var totalNumBytes: Long = 0L
     var totalNumOutput: Long = 0L
+    var maxNumWriters: Long = 0L
```

Comment:
should this just be 1 by default, as we'll never have 0 writers?

Reply:
It would be 0 for the GpuEmptyDirectoryDataWriter